### ISSUE TITLE
Remove dollar in auto-generated attributes

### ISSUE THREAD

**Issue by wilzbach** [microservices/microservice.guide#65]
![image](https://user-images.githubusercontent.com/4370550/55416079-79383100-556e-11e9-85d2-ac442a5fe632.png)

As this is not part of the markdown (e.g. https://github.com/microservices/microservice.guide/tree/master/schema/actions#arguments), I presume the dollar accidentally ended up in the output?

**Comment by judepereira** [comment]
@wilzbach It's intentional. We use it to describe something user defined, such as an action name.

**Comment by judepereira** [comment]
Also, these are not auto generated, but specified in the actual README.md files.

**Comment by wilzbach** [comment]
> It's intentional. We use it to describe something user defined, such as an action name.

A single dollar typically means end of stream. Using it for anything else is breaking conventions. Can't we at least do `${actionName}` then?

**Comment by judepereira** [comment]
`actions.${actionName}.http`is bash-ism. How about `actions.<action name>.http`?

**Comment by judepereira** [comment]
The `$` does work though - Steve used it initially, and then I understood what it meant by looking at it. The meaning is conveyed successfully :)

**Comment by wilzbach** [comment]
> How about actions.<action name>.http?

Of course. I prefer that even over `${actionName}`.
If we have to use a single symbol, use the established wildcard symbol `actions.*.http` (though this is a bit confusing with RegEx syntax) or the placeholder symbol `actions.•.http`.
tl;dr:

```
actions.<action name>.http
actions.•.http
```

> The meaning was conveyed successfully :)

But you're a very experienced programmer.
My point was that it is potentially confusing and that beginners will struggle to understand the meaning.

**Comment by judepereira** [comment]
I like `*` and `•`. I can switch the rendering of this for all in just one rule.

**Comment by williammartin** [comment]
$ also indicates a variable in a variety of languages, shell scripting, php, Perl etc which is where I suspect this came from, so I wouldn’t say it is breaking conventions. That said, I do agree that we can be far more descriptive here so +1.

**Comment by williammartin** [comment]
Ah I see that you were being explicit that a single dollar by itself would break conventions. Agree with that.
